packages/fxa-settings/src/components/Profile/index.tsx

```diff
@@ -22,7 +22,13 @@ export const Profile = () => {
       </h2>
 
       <div className="bg-white tablet:rounded-xl shadow">
-        <UnitRowWithAvatar />
+        <UnitRow
+          header="Picture"
+          headerValue={null}
+          avatar={avatar}
+          ctaText={avatar.url ? 'Change' : 'Add'}
+          route={`${HomePath}/avatar`}
```

Comment:
It looks like we currently have `${location.search}` tacked on to the end of this in `UnitRowWithAvatar`, I'd go ahead and include that here.

Reply:
We have a `${location.search}` at the end of `to` here https://github.com/mozilla/fxa/blob/21e80ce3192ddbd2b9ce6acf8ada9b5ea34f6bbe/packages/fxa-settings/src/components/UnitRow/index.tsx#L152-L156
so I didn't include it here. Should that be ok?

Reply:
Oh I see, great catch!